.env.example

```
@@ -6,6 +6,6 @@ LOG_PATH=/var/log
 DOCKER_SOCKET=/var/run/docker.sock
 CRON_PATHS=/etc/crontab,/etc/cron.d/*,/var/spool/cron/*,/var/spool/cron/crontabs/*,/var/spool/cron/tabs/*
 ALLOWED_ORIGINS=http://localhost:4200,http://127.0.0.1:4200,http://localhost:5173,http://127.0.0.1:5173
-COOKIE_SECURE=false  # set to true in production (requires HTTPS/reverse proxy)
+COOKIE_SECURE=true   # requires HTTPS / reverse proxy; set to false only for localhost dev
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The COOKIE_SECURE key should go before the CRON_PATHS key

(UnorderedKey)